### DISCUSSION TITLE
Connect to AWS RDS through a bastion host

### DISCUSSION THREAD

**Discussion by majagrubic** [prisma/prisma#17325] [Q&A]
I've been reading through this [thread](https://github.com/prisma/prisma/issues/5132), but I couldn't find an answer to my question.

I have a PostgreSQL database running on AWS RDS. The cluster isn't publicly available and requires a bastion host. I am able to connect to the DB through the bastion host using the `DBVisualizer` tool.

Now, I'd like to set up a project using Prisma as an ORM, but I can't figure out how to connect to AWS using bastion host. From the documentation and the thread above, I've managed to construct my URL string like this:

```
DATABASE_URL="postgresql://<username>:<password>@<cluster-endpoint>.us-east-1.rds.amazonaws.com:5432/postgres?schema=<schemaname>&sslcert=my-cert.pem&ssl_mode=require"
```

but of course, this won't work, since nowhere am I specifying the bastion host URL. How do I formulate the connection string using the bastion host?

I am currently getting an error: 
```
Error: P1011

Error opening a TLS connection: One or more parameters passed to a function were not valid.
```

**Comment by nurul3101** [comment]
Hi @majagrubic 👋 

I was able to find a related GitHub Issue: [#7048](https://github.com/prisma/prisma/issues/7048)
Can you have a look at the solution that worked for them, specifically this [comment](https://github.com/prisma/prisma/issues/7048#issuecomment-853780546)? They used AWS RDS with bastion host as well.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
